src/main.rs

```diff
@@ -57,8 +56,8 @@ impl ggez::event::EventHandler for State {
         // There are also other kyeboard and mouse functions, see:
         // https://docs.rs/ggez/0.5.1/ggez/event/trait.EventHandler.html
         match keycode {
-            KeyCode::W => { println!("W was pressed..."); }
-            _ => (), // Do nothing
+            KeyCode::W => println!("W was pressed..."),
+            _ => unimplemented!(),
```

Comment:
This function is called by the library for every key event. We should leave almost all keyboard events unprocessed; program should not panic for every unset key-press.
```suggestion
            _ => (),
```